build.xml

```diff
@@ -526,8 +526,8 @@
             <arg value="-q" />
             <arg value="org.apache.maven.plugins:maven-gpg-plugin:1.6:sign-and-deploy-file" />
             <arg value="-Dfile=@{file}" />
-            <arg value="-DgroupId=org.apache.cassandra" />
-            <arg value="-DartifactId=cassandra-parent" />
+            <arg value="-DgroupId=com.datastax.dse" />
```

Comment:
I thought `com.datastax.db` or `com.datastax.dse.db`

Reply:
WFM
@JeremiahDJordan ?

Reply:
I think this is fine for now.  We will probably have a discussion again about such things when we are pushing things publicly.